src/Components/Common/Sidebar/Sidebar.tsx

```diff
@@ -101,7 +104,7 @@ const StatelessSidebar = ({
     } else {
       indicatorRef.current.style.display = "none";
     }
-  }, [activeLink]);
+  }, [activeLink, lastIndicatorPosition]);
```

Comment:
updating `lastIndicatorPosition` and having it in deps array might cause infinite loop